packages/api/lib/response.js

```diff
@@ -35,28 +33,28 @@ function getToken(req) {
 }
 
 
-function resp(context, err, body, status = null, headers = null) {
+function resp(context, err, bodyArg, statusArg = null, headers = {}) {
```

Comment:
While you're in here, you should document this function.

Reply:
We'd be better off refactoring so this function isn't used.  It uses `context.succeed` to indicate success, which is no longer documented by AWS.  There are two other ways (using a callback or returning a Promise) that should be used with Lambda functions instead.